src/components/files/deleteFiles.ts

```diff
@@ -9,27 +9,27 @@ const { filesFolder, genericThumbnailsFolderName } = config
 export const filesPath = path.join(getAppEntryPointDir(), filesFolder)
 
 export interface FileDetail {
-  id: number
-  uniqueId: string
-  originalFilename: string
+  originalFilename?: string
   filePath: string
-  thumbnailPath: string
+  thumbnailPath?: string
 }
 
 export const deleteFile = async (file: FileDetail) => {
   const { filePath, thumbnailPath, originalFilename } = file
   try {
     await fsPromises.unlink(path.join(filesPath, filePath))
     // Don't delete generic (shared) thumbnail files
-    if (!thumbnailPath.match(genericThumbnailsFolderName))
+    if (thumbnailPath && !thumbnailPath.match(genericThumbnailsFolderName)) {
```

Comment:
Modified "deleteFile" method to handle a missing "thumbnailPath" -- this was the source of the error you were getting.